spec/lib/salus_spec.rb

```diff
@@ -172,5 +173,30 @@
         end
       end
     end
+
+    context 'With plugins' do
+      it 'Should update config based on plugin' do
+        Dir.chdir('spec/fixtures/blank_repository2') do
+          plugin_dir = "../../../lib/salus/plugins"
+          FileUtils.cp('test_plugins/my_filter.rb', plugin_dir + '/my_filter.rb')
```

Comment:
Thanks for including the spec 👍 

One thought here, if the logic for the directory was placed into a method you could do away with the file copying / deleting here and just stub the value:

`expect(Salus::PluginManager).to receive(:plugin_directory).and_return('spec/fixtures/blank_repository2/test_plugins')`

Reply:
updated accordingly